daily_spellbook/dbt_project.yml

```diff
@@ -0,0 +1,91 @@
+#Welcome to your Spellbook!
+
+name: "daily_spellbook"
+version: "1.0.0"
+config-version: 2
+
+quoting:
+  database: false
+  schema: false
+  identifier: false
+
+# This setting configures which "profile" dbt uses for this project.
+# profile: "spellbook-poc-tokens"
+profile: "spellbook-local"
+
+vars:
+  DBT_ENV_CUSTOM_ENV_S3_BUCKET: "{{ env_var('DBT_ENV_CUSTOM_ENV_S3_BUCKET', 'local') }}"
+  DBT_ENV_INCREMENTAL_TIME: "{{ env_var('DBT_ENV_INCREMENTAL_TIME', '1') }}"
+  DBT_ENV_INCREMENTAL_TIME_UNIT: "{{ env_var('DBT_ENV_INCREMENTAL_TIME_UNIT', 'day') }}"
+
+# These configurations specify where dbt should look for different types of files.
+# You don't need to change these!
+model-paths: ["models", "../sources"]
+analysis-paths: ["analyses"]
+test-paths: ["tests"]
+seed-paths: ["seeds"]
+macro-paths: ["../macros"]
+snapshot-paths: ["snapshots"]
+
+target-path: "target" # directory which will store compiled SQL files
+clean-targets: # directories to be removed by `dbt clean`
+  - "target"
+  - "dbt_packages"
+
+# Configuring tests
+# Full documentation: https://docs.getdbt.com/reference/test-configs
+tests:
+  daily_spellbook:
+    +store_failures: true  # store failures for all tests
+
+# Configuring seeds
+# Full documentation: https://docs.getdbt.com/reference/seed-configs
+# For configuring individual seeds (e.g. overriding column types) we recommend a yml file nested in the same folder as the seed
+# example: seeds/cryptopunks/schema.yml
+# TODO: uncomment when seeds are added
+# seeds:
+#   daily_spellbook:
+#     +schema: test_data
+
+# defining search order of macro invokation
+dispatch:
+  - macro_namespace: dbt_utils
+    search_order: ['trino_utils', 'dbt_utils']
+
+# Configuring models
+# Full documentation: https://docs.getdbt.com/docs/configuring-models
+models:
+  daily_spellbook:
+    +post-hook:
+      - sql: "{{ set_trino_session_property(is_materialized(model), 'writer_scaling_min_data_processed', model.config.get('writer_min_size', '500MB')) }}"
+        transaction: true
+      - sql: "{{ set_trino_session_property(is_materialized(model), 'task_scale_writers_enabled', false) }}"
+        transaction: true
+      - sql: "{{ optimize_spell(this, model.config.materialized) }}"
+        transaction: true
+      - sql: "{{ mark_as_spell(this, model.config.materialized) }}"
+        transaction: true
+    +materialized: view
+    +schema: no_schema    # this should be overriden in model specific configs
+    +view_security: invoker
+
+    chainlink:
+      +schema: chainlink
+      arbitrum:
+        +schema: chainlink_arbitrum
+      avalanche_c:
+        +schema: chainlink_avalanche_c
+      base:
+        +schema: chainlink_base
+      bnb:
+        +schema: chainlink_bnb
+      ethereum:
+        +schema: chainlink_ethereum
+      fantom:
+        +schema: chainlink_fantom
+      gnosis:
+        +schema: chainlink_gnosis
+      optimism:
+        +schema: chainlink_optimism
+      polygon:
+        +schema: chainlink_polygon
```

Comment:
just noting that we will want to remove these soon and put in models. just a lot of models to do. we can wait until the team refactors the code into macros (already underway)

Reply:
Yeah agree, was surprised to see these in the project file